tests/test_cli.py

```diff
@@ -1198,6 +1198,14 @@ def add_compare(self, cat, args, compbase, **kwargs):
 
 
 
+#####################
+# loongarch64 tests #
+#####################
+
+c.add_compare("--osinfo fedora19 --nodisks --pxe --connect " + utils.URIs.kvm_loongarch64, "kvm-loongarch64-basic")
```

Comment:
@lixianglai I just noticed this but I think instead of using `utils.URIs.kvm_loongarch64` directly here you should define `URI-KVM-LOONGARCH64` like other architectures.

In terms of test coverage, the choice of having PXE only is a bit peculiar, and the use of `--nodisks` in particular means that we don't see whether the choice of disk model/controller would be the correct one. I would just copy and adapt the riscv64 tests, which ensure that both the headless and graphics scenarios are working as intended. Just use SPICE instead of VNC to ensure that sound is present.

Reply:
@lixianglai any chance you could take a look at this? A new virt-manager release might happen relatively soon, and it would be a shame if loongarch64 support couldn't make it in. Thanks!

Reply:
I am very sorry for the late reply. Because I am busy with other work recently, I did not pay attention to the email message. I will modify it according to the opinion within this week, thank you very much!

Reply:
Hi Andrea, I updated the loongarch64 test case with reference to the riscv64 test case on the latest code